.github/workflows/ci-pull-request.yaml

```diff
@@ -33,8 +33,15 @@ jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        example:
+          - "secure_threat_detection/single/main.tf"
```

Comment:
For this PR, we are sticking to testing single TF module for Threat Detection & CSPM (due to the problems you mentioned which we need to resolve). am I right?

Reply:
In this PR, both modules `event-bridge` and `trust-relationship` have passing functional tests.